disco/client.py

```diff
@@ -122,7 +122,7 @@ def update_presence(self, status, game=None, afk=False, since=0.0):
         since : float
             How long the client has been afk for (in seconds).
         """
-        if game and not isinstance(game, Game):
+        if game and not isinstance(game, Activity):
             raise TypeError('Game must be a Game model')
```

Comment:
Error should reflect this change, something like "Activity must be an Activity model"

Reply:
Smart! I couldn't read another line down, mostly because my IDE didn't tell me. Thanks for pointing this out so I can fix it!